x-pack/metricbeat/module/aws/s3_daily_storage/s3_daily_storage.go

```diff
@@ -38,25 +37,18 @@ func init() {
 // interface methods except for Fetch.
 type MetricSet struct {
 	*aws.MetricSet
-	logger *logp.Logger
 }
 
 // New creates a new instance of the MetricSet. New is responsible for unpacking
 // any MetricSet specific configuration options if there are any.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	cfgwarn.Beta("The aws s3_daily_storage metricset is beta.")
-	s3Logger := logp.NewLogger(aws.ModuleName)
 
 	moduleConfig := aws.Config{}
 	if err := base.Module().UnpackConfig(&moduleConfig); err != nil {
 		return nil, err
 	}
 
-	if moduleConfig.Period == "" {
```

Comment:
How is this change related?

Reply:
Not related. Sorry forgot to update on the PR . This check is not needed here because it's already done in `aws.NewMetricSet(base)` in line 60.